gensim/models/doc2vec.py

```diff
@@ -142,16 +173,33 @@ def train_document_dm(model, doc_words, doctag_indexes, alpha, work=None, neu1=N
                        model.vocab[w].sample_int > model.random.rand() * 2**32]
 
         for pos, word in enumerate(word_vocabs):
-            reduced_window = model.random.randint(model.window)  # `b` in the original doc2vec code
+            reduced_window = model.random.randint(
+                model.window)  # `b` in the original doc2vec code
             start = max(0, pos - model.window + reduced_window)
-            window_pos = enumerate(word_vocabs[start:(pos + model.window + 1 - reduced_window)], start)
-            word2_indexes = [word2.index for pos2, word2 in window_pos if pos2 != pos]
-            l1 = np_sum(word_vectors[word2_indexes], axis=0) + np_sum(doctag_vectors[doctag_indexes], axis=0)
+            window_pos = enumerate(
+                word_vocabs[
+                    start:(
+                        pos +
+                        model.window +
+                        1 -
+                        reduced_window)],
+                start)
+            word2_indexes = [
+                word2.index for pos2,
+                word2 in window_pos if pos2 != pos]
```

Comment:
This split seems absurd to me: a line-break in the MIDDLE of a tuple-desctructuring?!? Wasn't the original fairly idiomatic as a 1-liner (even though slightly over 79 chars)?